utils/utest.yaml

```diff
@@ -48,6 +48,10 @@
   tests:
     - cmd: ["src/gurt/tests/test_gurt"]
     - cmd: ["src/gurt/tests/test_gurt_telem_producer"]
+- name: DTX
+  base: "PREFIX"
+  tests:
+    - cmd: ["bin/dtx_tests"]
```

Comment:
if you're installing them, you will want to update the daos.spec file.   Unless you have a really good reason that these need to be installed, I'd recommend running from build directory as with many other tests.

Reply:
I believe in regards it allows testing local VOS transactions it should be installed whenever `vos_tests` is.

Done.

Reply:
yeah, we aren't very consistent with what we do and don't install.

Reply:
no, it's something we need to address at some point to ensure we are consistent.